### DISCUSSION TITLE
License issue

### DISCUSSION THREAD

**Discussion by Showmaster3000** [Tyrrrz/Onova#49] [General]
Hi, thank you for your nice library.
Just want to use it, but the license on github and nuget differ. Github page says _MIT license_ where nuget page says _LGPL-3.0-only_

To check:
- open https://www.nuget.org/packages/Onova and see "[LGPL-3.0-only](https://licenses.nuget.org/LGPL-3.0-only) license" in the info.
- open https://github.com/Tyrrrz/Onova and see  "MIT license" in the info and "[License.txt](https://github.com/Tyrrrz/Onova/blob/master/License.txt)"

The question is: what is correct?

**Comment by Tyrrrz** [comment]
The latest version of the package is LGPL-3.0, while the current code in the repo is MIT. The license was changed since the last version of the package was released, but there wasn't an update since then to reflect that on NuGet.

**Reply by Tyrrrz** [reply]
I've deployed a new version of the package to sync the licenses
